akka-javasdk/src/main/scala/akka/javasdk/impl/AnySupport.scala

```diff
@@ -200,6 +203,97 @@ private[akka] object AnySupport {
     }
 
   def extractBytes(bytes: ByteString): ByteString = bytesToPrimitive(BytesPrimitive, bytes)
+
+  def toScalaPbAny(bytesPayload: BytesPayload): ScalaPbAny = {
+    if (bytesPayload.contentType.startsWith(DefaultTypeUrlPrefix))
+      ScalaPbAny(
+        typeUrl = bytesPayload.contentType,
+        value = ByteStringUtils.toProtoByteStringUnsafe(bytesPayload.bytes))
+    else
+      ScalaPbAny(
+        typeUrl = bytesPayload.contentType,
+        value = encodeByteArray(ByteStringUtils.toProtoByteStringUnsafe(bytesPayload.bytes)))
+  }
+
+  private def encodeByteArray(bytes: ByteString): ByteString = {
```

Comment:
This is a little confusing. I know this code comes from the runtime. Should we assume that is temporary? Because I was expecting the SDK to only know what a `BytesPayload` is.

Reply:
Anyway, so suggestions/questions about this method and it's usage just above.

Better if `toProtoByteStringUnsafe` is responsible for adding the proto tag. 
Or this `encodeByteArray` should be called `addProtoTag` or something like that.